README.md

```diff
@@ -1,9 +1,10 @@
-
-# openui5-preload
+# ui5-preload
```

Comment:
@CarlosOrozco88 can you revert the rename in `README.md` and the mention of the fork. Otherwise, looks good to me.